### ISSUE TITLE
New Number Registration Not working

### ISSUE THREAD

**Issue by rajenpal033** [venomous0x/WhatsAPI#1090]
New number registration not working now. It's showing and exception, there is a problem requesting the code, while same thing was working two days ago. @shirioko can you please check for that.

**Comment by waichoi** [comment]
Yes, I have the same issue, I just download and use the latest WART-1.7.3.0 exe version.   
The previous 1.7.2.0 version is working too 5 days ago.   Mean I am sure my procedure are correct base on my 6 days ago experience.
